hztb-aws/src/main/java/com/svs/hztb/aws/client/AWSClientProcessor.java

```diff
@@ -46,8 +44,11 @@ public String execute(byte[] bytes, String type, String name) {
 			fileName = productBucketName + SLASH + name + ".jpg";
 		}
 
-		awsConfigurationProvider.getAmazonS3Client().putObject(new PutObjectRequest(hztbBucketName, fileName, bis, omd)
-				.withCannedAcl(CannedAccessControlList.PublicRead));
+		/*
+		 * awsConfigurationProvider.getAmazonS3Client().putObject(new
+		 * PutObjectRequest(hztbBucketName, fileName, bis, omd)
+		 * .withCannedAcl(CannedAccessControlList.PublicRead));
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://ec2-52-23-152-124.compute-1.amazonaws.com:9000/coding_rules#rule_key=squid%3ACommentedOutCodeLine)